### PR TITLE
Package.json

### PR DESCRIPTION
**Pull request by JakeSidSmith**
Preparing for NPM.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "reorder",
+    "sort",
+    "reorderable",
+    "sortable",
+    "drag and drop",
+    "mobile",
+    "touch"
+  ],
+  "dependencies": {
+    "react-tools": "~0.12.1"
```
Comment (weargoggles) [inline package.json]: Reading the repo page for react, it does seem as if you could get away with `"react": "~0.12"` here

Comment (JakeSidSmith) [inline package.json]: I can go for that. ^_^

Comment (JakeSidSmith) [inline package.json]: @weargoggles Does this include the tools though? https://www.npmjs.com/package/react

Comment (weargoggles) [inline package.json]: I missed the small amount of JSX in there. I suppose then react-tools is a devDependency rather than a dependency!

Comment (JakeSidSmith) [inline package.json]: Will change to `"react-tools": "~0.12"`

**Pull request merged**